addons/mail/static/src/components/chat_window_header/chat_window_header.xml

```diff
@@ -72,7 +72,7 @@
                             <i class="fa fa-expand"/>
                         </div>
                     </t>
-                    <t t-if="!props.hasCloseAsBackButton">
+                    <t t-if="!chatWindow.hasCloseAsBackButton">
```

Comment:
It's usually a bad practice to test the negative to display something, would be more ideal to have "has close button" here, and "has back button" in the other case, not really the same variable (even if in practice for now, their compute will make them mutually exclusive).

Or just have a "selection" kind of field with 3 choices (as string) instead if we really want to show they are mutually exclusive: either of them, or nothing.

Reply:
The feature was designed with close button first, and then have back button being a replacement of close button visual for mobile. So this field has just been introduced that way, hence the negative assertion.

I disagree with the statement that "it usually a bad practice to test the negative to display something".
It's a problem when the specs are poor and results in `undefined` being a value treated as basically `false` accidentally (as `undefined ` is falsy in JS).

Here the field is clearly `true` or `false`. And the default value is clearly a boolean.
So there is absolutely no problem with negative conditional statement at all.
It's really a PoV, and the boolean values could have been switched (`true`/`false` <=> `false`/`true`), as long as the logical statement in the specs make sense (of course with appropriate wording on chosen PoV).

Reply:
But the problem is not that, the problem is that only one boolean is controlling 2 features. There are 2 different parts in the template, so there should be 2 fields, one controlling each.

Reply:
The 2 parts are conceptually disjointed, so there really is no reason to have dedicated fields for each part.